src/app/public/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -131,6 +140,14 @@ export class SkyDatepickerInputDirective implements
     }
   }
 
+  public ngAfterViewInit(): void {
```

Comment:
@Blackbaud-SteveBrush So I have an implementation of this locally, but are you sure we want to only set the value when it is valid?
Right now we do support people inserting values such as `abcdef` and letting the form be marked as invalid. I would prefer the new way of just blocking invalid values, but am concerned that it could be considered breaking